docs/modules/release-notes/pages/5-4-0.adoc

```diff
@@ -0,0 +1,248 @@
+https://github.com/hazelcast/hazelcast/pull/25154[#25154]
+
+=== Serialization
+
+* Added the ability to use consistent serialization service for `ByKey` plans.
+https://github.com/hazelcast/hazelcast/pull/25631[#25631]
+
+=== WAN Replication
+
+* Improved dead connection handling for WAN replication static IP discovery, by introducing health check to the discovery strategy.
+https://github.com/hazelcast/hazelcast/pull/25364[#25364]
+* Added support of dynamically adding WAN replication configurations using Java API.
+https://github.com/hazelcast/hazelcast/pull/25118[#25118]
+* Added support of evicting map and cache entries through WAN replication by introducing two new properties. When enabled, WAN replication
+events are fired in case values are evicted from the map and cache objects. See xref:wan:tuning.adoc#replicating-imap-icache-evictions[Replicating `IMap`/`ICache` Evictions].
+https://github.com/hazelcast/hazelcast/pull/24941[#24941]
+
+=== Connectors
+
+* Removed the beta annotations from the MongoDB classes.
+https://github.com/hazelcast/hazelcast/pull/25743[#25743]
+* Added TLS support for MongoDB data connections.
+https://github.com/hazelcast/hazelcast/pull/25301[#25301]
+* Added support of inline Avro schemas for Kafka mappings.
+https://github.com/hazelcast/hazelcast/pull/25207[#25207]
+* Added support of `DataSerializable` serialization to Mongo connector.
+https://github.com/hazelcast/hazelcast/pull/25197[#25197]
+* Check for existing resources for Mongo connector is now done only once; previously, it was performed on every processor.
+https://github.com/hazelcast/hazelcast/pull/24953[#24953]
+* Added support of Microsoft SQL server for Hazelcast's JDBC connector.
+https://github.com/hazelcast/hazelcast/pull/25071[#25071]
+* Added the ability to configure the pool size of a MongoDB data connection. See xref:sql:mapping-to-mongo.adoc#creating-a-mongodb-mapping[Creating a MongoDB Mapping].
+https://github.com/hazelcast/hazelcast/pull/25027[#25027]
+
+=== Metrics
+
+* Improved the naming convention for CP Session, Lock, and Persistence metrics.
+https://github.com/hazelcast/hazelcast/pull/24843[#24843], https://github.com/hazelcast/hazelcast/pull/24836[#24836]
+
+=== Licensing
+
+* License keys are now being masked in the license expiration notifications.
+https://github.com/hazelcast/hazelcast/pull/24800[#24800]
+
+=== Data Structures
+
+* Added check for negative positions on the collections' `getItemAtPositionOrNull()` method.
+https://github.com/hazelcast/hazelcast/pull/25672[#25672]
+* Added the `IMap.localValues()` and `IMap.localValues(Predicate)` methods for a faster access to locally owned values in maps.
+https://github.com/hazelcast/hazelcast/pull/24673[#24763]
+* Added the `deleteAsync()` for maps to asynchronously remove a provided map entry key.
+https://github.com/hazelcast/hazelcast/pull/23509[#23509]
+
+=== Other Enhancements
```

Comment:
what is the difference between the "Enhancements" title and this "Other Enhancements" title?